common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -114,6 +114,18 @@ class CapaFields(object):
         help=_("Amount of time after the due date that submissions will be accepted"),
         scope=Scope.settings
     )
+    show_correctness = String(
+        display_name=_("Show Correctness"),
+        help=_("Defines when to show whether a learner's answer to the problem is correct. "
+               "Configured on the subsection.  A default value can be set in Advanced Settings."),
```

Comment:
Does this setting (this is in an individual problem, correct?) override the subsection setting? If so, I would change the second sentence to "This setting overrides the setting for the subsection."

If we keep the third sentence ("A default value..."), include only one space before the beginning of the sentence.